lib/iris/tests/experimental/test_animate.py

```diff
@@ -40,6 +40,7 @@
 @tests.skip_plot
 class IntegrationTest(tests.GraphicsTest):
     def setUp(self):
+        super(IntegrationTest, self).setUp()
```

Comment:
@bjlittle do we need to follow this pattern for _all_ instances of test classes inheriting from `tests.GraphicsTest`? If so, you'll also need to update the following:

* In `lib/iris/tests`:
 * `test_analysis.TestRotatedPole`
 * `test_coordsystem.Test_LambertConformal`
 * `test_mapping.TestBasic`
 * In `testPlot`: `Test1dPlotMultiArgs`, `TestMissingCoord`, `TestMissingCS`, `TestAttributePositive`, TestPlotOtherCoordSystems
* All affected test classes in `lib/iris/tests/integration/test_grib_load.py`
* Test class in `lib/iris/tests/integration/test_netcdftime.py`